mirrord/agent/env/src/envs.rs

```diff
@@ -51,3 +51,15 @@ pub const POD_IPS: CheckedEnv<Vec<IpAddr>> = CheckedEnv::new("MIRRORD_AGENT_POD_
 ///
 /// Should follow `tracing` format, e.g `mirrord=trace`.
 pub const LOG_LEVEL: CheckedEnv<String> = CheckedEnv::new("RUST_LOG");
+
+/// Container id of the target we're attaching to, e.g. `mirrord exec -t
+/// pod/glorious-cat/container/[cat-container]`, this is the id of `cat-container` that you
+/// can retrieve with `kubectl describe glorious-cat`.
+///
+/// For `Mode::Ephemeral`, this is also used to get the target's `pid`. When this is not
```

Comment:
is it used for other thing other than `Mode::Ephemeral` ? if not, "this is used"

Reply:
Changed.

Reply:
The "For Mode::Ephemeral" still might make it seem like it's used in other places, but I'm failing to come up with a phrase that says it's only used in that mode.